### PR TITLE
Added Publicationslist

### PR DESCRIPTION
**Pull request by Smokrow**

### CODE REVIEW

File: wg-iot-edge/README.md [hunk 1]
```diff
@@ -47,5 +47,7 @@ are then managed by the appropriate SIGs based on impacted areas of code.
 * Evaluate and extend existing CLI tools to manage k8s clusters running in remote edge locations.
 
 It is an open and active working group, and we always welcome new additions!
+## Publications
```
Comment (nikhita) [inline wg-iot-edge/README.md]: nit: can you add a newline before the heading?

File: wg-iot-edge/README.md [hunk 2]
```diff
@@ -47,5 +47,7 @@ are then managed by the appropriate SIGs based on impacted areas of code.
...
+* [Edge security challenges](whitepapers/edge-securuity-challenges/)
```
Comment (nikhita) [inline wg-iot-edge/README.md]: Can you also correct the main name while we are here? s/securuity/security

Comment (Smokrow) [reply wg-iot-edge/README.md]: Right now the blog post from CNCF links to this place. I dont know if we should change it and break the link

**Comment by Smokrow** [comment]
Sorry for the late commits. I hope this is fine

**Comment by dejanb** [comment]
@Smokrow LGTM. I'll take a look if we can rename file and keep the old link. Can you just squash these two commits?

**Comment by dejanb** [comment]
>I'll take a look if we can rename file and keep the old link.

Unfortunately I don't think that's possible, so we'll need to live with this typo

**Comment by mrbobbytables** [comment]
> Right now the blog post from CNCF links to this place. I dont know if we should change it and break the link

> @Smokrow LGTM. I'll take a look if we can rename file and keep the old link. Can you just squash these two commits?

I'd give the CNCF a ping, they should be able to update the link pretty quick 👍

**Comment by dejanb** [comment]
@mrbobbytables Thanks. I'll ping them. We can fix it in the follow-up PR

**Comment by Smokrow** [comment]
@dejanb I squashed the commits

**Pull request merged**